ci-operator/step-registry/ibmcloud/deprovision/bastionhost/ibmcloud-deprovision-bastionhost-commands.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+function run_command() {
+    local CMD="$1"
+    echo "Running Command: ${CMD}"
+    eval "${CMD}"
+}
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+    "${IBMCLOUD_CLI}" login -a https://cloud.ibm.com -r ${LEASED_RESOURCE} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
```

Comment:
Why we need -a parameter here?

Reply:
https://cloud.ibm.com/docs/cli?topic=cli-ibmcloud_cli#login-options, I think `-a https://cloud.ibm.com/` is default value, I will talk with Christopher J Schaefer whether this is required.

Reply:
Shouldn't ibmcloud cli set the default value, if we set it via `-a` option, dose that mean ibmcloud is not reliable?